OpenNept4une.sh

```diff
@@ -287,6 +306,20 @@ install_feature() {
     # Proceed if the user agrees or if auto_yes is true
     if [[ $user_input =~ ^[Yy]$ || -z $user_input || $auto_yes = "true" ]]; then
         echo -e "Running $feature_name Installer...\n"
+
+        if [ "$feature_name" = "Updated Display Firmware" ]; then
+            if [ -d "${HOME}/display_firmware" ]; then
+                echo "Display_Firmware installed. Proceeding with installer!"
+            else
+                echo "Display_Firmware not installed. Downloading"
+                mkdir display_firmware
+                git clone --filter=blob:none --sparse https://github.com/OpenNeptune3D/display_firmware.git "${HOME}/display_firmware"
```

Comment:
What `--filter=blob:none --sparse` brings to the table? Just curious

Reply:
I'm okay with it as long as we merge it on `dev` for a test run, hopefully it should be able to be updated with a `git pull` to bring the latest stuff from `display_firmware`. If something goes wrong it is an easy fix anyways :)